tests/unit_tests/models/test-environment.php

```diff
@@ -18,6 +18,90 @@ public function __construct() {
   }
 
   /**
+   * Ensure correct connection info for development environments
+   * Development environment connection info includes git parameters
+   *
+   * @return void
+   *
+   * @vcr site_connection-info
+   */
+  public function testConnectionInfoDev() {
+    logInWithBehatCredentials();
+    $site     = $this->sites->get('behat-tests');
+    $test_env = $site->environments->get('dev');
+    $info     = $test_env->connectionInfo();
+
+    // SFTP Connection Info
+    $sftpInfo = [
```

Comment:
PSR-2 allows underscores in variable names, but most PSR-2 code uses camelCase.  (https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md has an example variable `$longArgs_longVars`)